docs/gallery/sections/writing_workflows_with_workgraph/autogen/GALLERY_HEADER.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+================================
+Running workflows with WorkGraph
+================================
```

Comment:
sphinx-gallery requires a `GALLERY_HEADER.rst` but we do not use if anywhere therefore the `:orphan:` at the top to surppres warnings